Objects/TI4_Helpers/TI4_UnitHelper.ttslua

```diff
@@ -804,6 +804,30 @@ local _unitModifiers = {
         end
     },
 
+    ['Raid Formation'] = {
+        description = 'ANTI-FIGHTER BARRAGE hits in excess of fighter targets cause SUSTAIN DAMAGE units to become damaged.',
+        owner = OWNER.SELF,
+        type = TYPE.MUTATE,
+        apply = function(unitAttrs, params)
+            -- This is called by automatic-fill-multiroller tool (with opponent ship context),
+            -- then later it is called again by multiroller without opponent context. To fix
+            -- this, explore injecting opponent-unitType-to-Count and using that in multiroller.
+
+            -- Ensure we have some opponent data, this 'apply' requires that minimum context.
+            if not params.opponentUnitTypeToCount then
```

Comment:
I'm not sure I like the idea of the roller having more abilities when used with auto-fill (vs manually filling it in), but I see the value here.

Reply:
It is awkward, and not a good precedent. I could see just dropping this, honestly. I think there's a lot of value in keeping the roll result report simple.

I figure it is worth seeing in play once or twice, though.

Reply:
Works for me, try it out and pull it if deemed too much.